city_scrapers/spiders/chi_northwest_home_equity.py

```diff
@@ -0,0 +1,375 @@
+        # Northwest Home Equity Assurance Program is governing commission appointed by the mayor
+        return COMMISSION
+
+    def _parse_start(self, item, isPast, response):
+        """Parse start datetime as a naive datetime object."""
+        """
+        Finding the start datetime is tricky for past meetings.
+        The website doesn't have a constant way of telling date/time.
+        Our methodology uses the meeting's date to search for the meeting in
+        the past meeting sidebar and fetch the time.
+        """
+        if isPast:
+            # sometimes the date is in the title of that Meet Minutes
+            title = item.xpath(".//h2[@class='entry-title']/a/text()").extract()
+            date = title[0].split(" ")[0:3]
+
+            monthNumberString = date[0].lower()
+            monthNumberString = self.calendar.get(monthNumberString)
+            dayString = date[1].replace(",", "")
+            yearString = date[2]
+
+            # if the string fetched is not a month then search for it in the description
+            if not monthNumberString:
+                description = item.xpath("./div[@class='entry-content']/p/text()").get()
+                description = description.split(" ")
+                monthNumberString = ""
+
+                # check each word in the description if it is a month
+                for i in range(len(description)):
+
+                    if self.calendar.get(description[i].lower()):
+                        monthNumberString += self.calendar[description[i].lower()]
+
+                        # there are different formatting of month, day, and year
+                        # the day could be before or after the year
+                        if (len(description[i + 1]) >= 4):
+                            dayString = description[i - 1].replace(",", "")
+                            yearString = description[i + 1][:4]
+
+                        else:
+                            dayString = description[i + 1].replace(",", "")
+                            yearString = description[i + 2][:4]
+
+            # in case the day is only a single digit
+            if len(dayString) == 1:
+                dayString = "0" + dayString
+
+            # create the complete date string
+            dateStringFourDigit = "/".join([monthNumberString, dayString, yearString])
+
+            queryString = "//ul[li='" + dateStringFourDigit + "']/li[2]/text()"
+
+            # search the time in the Past Meeting sidebar
+            timeString = response.xpath(queryString).get()
+
+            # not found then default to 12am
+            if not timeString:
+                timeString = "12:00:00AM"
+            else:
+                timeString = timeString.split(" ")
+                timeString = timeString[0] + ":00" + timeString[1].upper()
+
+            dateString = dateStringFourDigit + " " + timeString
+            datetimeFormat = "%m/%d/%Y %H:%M:%S%p"
```

Comment:
If you're using `%p` here then hours should be pulled with `%I` instead of `%H`

Reply:
Thanks, didn’t know that.